content/overview/overview/how-to-purchase.md

```diff
@@ -8,8 +8,8 @@ weight: 20
 
 ## How can I purchase cloud.gov?
 
-Cloud.gov services can only be purchased by government agencies and currently we limit our business to federal agencies, though we hope to provide services to state and local agencies in the future. If you represent an agency that wants to use cloud.gov services, please contact us by filling out [our initial intake form] (https://docs.google.com/forms/d/e/1FAIpQLSevZfuJ_4KE-MZlm9gttYfsXQp0PJL7OR6k6LbZ9XnFn-oA6g/viewform). After we receive your form, we'll schedule a follow conversation with you to learn more about your needs and how we can help. 
+cloud.gov services can only be purchased by government agencies. Currently, cloud.gov’s business is limited to federal agencies, though we hope to provide services to state and local agencies in the future. If you represent an agency that wants to use cloud.gov services, please contact us by filling out [our initial intake form] (https://docs.google.com/forms/d/e/1FAIpQLSevZfuJ_4KE-MZlm9gttYfsXQp0PJL7OR6k6LbZ9XnFn-oA6g/viewform). After we receive your form, we’ll schedule a follow conversation with you to learn more about your needs and how we can help.
 
-Note that we rely on the Interagency Agreement ("IAA") process to sell our services to other agencies. Currently, the process takes, on average, 6-8 weeks to process (we are working to make this faster). If you would like to learn more about the IAA process and the information that you need to have ready to begin the process, please visit [18F's page about this] (https://pages.18f.gov/iaa-forms/primer.html).
+cloud.gov services are sold through 18F, which relies on the Interagency Agreement ("IAA") process to sell to other agencies. Currently, the process takes, on average, 6-8 weeks to process (we are working to make this faster). If you would like to learn more about the IAA process and the information that you need to have ready to begin the process, please visit [18F's page about this] (https://pages.18f.gov/iaa-forms/primer.html).
```

Comment:
"18F sells cloud.gov services to other agencies through the Interagency Agreement ("IAA") process." ?